HW_3/task_1.py

```diff
@@ -0,0 +1,23 @@
+# Задача 16: Требуется вычислить, сколько раз встречается некоторое число X в массиве A. 
+# Пользователь в первой строке вводит натуральное число N – количество элементов в массиве. 
+# В последующих  строках записаны N целых чисел Ai (можно использовать псевдогенерацию). 
+# Последняя строка содержит число X.
+# *Пример:*
+# 5
+# 7 -2 3 5 1
+# 3
+# -> 1
+
+import random
+
+countEl = int(input('Input count of elements: '))
+lst = [random.randint(1, countEl) for i in range(countEl)]
+print(*lst)
+findEl = int(input('Input number for search: '))
+for el in lst:
+    if el == findEl:
+        print(lst.count(el))
+        break
+    else: 
+        print('Number not found')
+        break
```

Comment:
А зачем вам тут вообще цикл, если он будет делать всего 1 итерацию?
```suggestion
print(lst.count(findEl))
```
Вот и все решение. Или можно сделать так, использовав моржовый оператор, если хотите оставить сообщение, что элемента нет
```suggestion
if res := lst.count(findEl):
    print(res )
else: 
    print('Number not found')
```